modules/tensor_mechanics/include/materials/ComputeCappedWeakPlaneStress.h

```diff
@@ -7,7 +7,7 @@
 #ifndef COMPUTECAPPEDWEAKPLANESTRESS_H
```

Comment:
done ComputeCappedWeakPlaneStress -> CappedWeakPlaneStressUpdate.  Also similar for ComputeCappedWeakInclinedPlaneStress.  Also similar for TwoParameterPlasticity -> TwoParameterPlasticityStressUpdate